include/swift/ABI/MetadataValues.h

```diff
@@ -748,6 +748,14 @@ class ConformanceFlags {
     IsConformanceOfProtocolMask = 0x01u << 18,
     HasGlobalActorIsolation = 0x01u << 19,
 
+    // NOTE: This is currently specialized to detecting a non-default
+    // implementation of the 'SerialExecutor/isIsolatingCurrentContext' method,
+    // however the pattern of detecting "has non-default impl of ..." may be
+    // generalized if we would like it to; we could store another trailing object
+    // after the HasGlobalActorIsolation one, and use that to record which
+    // requirement has the non-default implementation that we need to special handle.
+    HasNonDefaultSerialExecutorIsIsolatingCurrentContext = 0x01u << 20,
```

Comment:
I don't think we need to do this. When we start running out of bits, we can use the last bit to add a trailing "flags", like we've done with various other data structures. To that end, please have the comment here say what this bit *is*, not what we could use it for.

Reply:
Great, will do!